components/Prize/PrizeItem.scss

```diff
@@ -0,0 +1,84 @@
+@import 'topcoder/tc-includes';
+
+.PrizeItem {
+  display:inline-block;
+  height: 20px;
+  margin-right: 15px;
+    
+  span {
+    font-family: "Roboto-Medium", Arial, Helvetica, sans-serif;
+    font-size: 13px;
+    line-height: 15px;
+    color: $white;
+    font-weight: 500;
+  }
+  .PrizeCircle{
+    display: inline;
+    padding: 1px 4px 1px 4px;
+    min-width: 15px;
+    max-height: 15px;
+    background: #C3C3C8;
+    border-radius: 8px;
+    text-align: center;
+    font-size: 11px;
+    line-height: 13px;
+    font-family: "Roboto-Black", Arial, Helvetica, sans-serif;
+    font-weight: bold;
+    color:  #47474F;
+  }
+ 
+}
+
+.Prize {
+  .PrizeItem:nth-child(1) {
+    .PrizeCircle{
+      background: #FFD900;
```

Comment:
Please use variables for colors. If you don't find them in _tc-colors.scss (referenced from appirio-styles dependency), please put a variable in the top of the component's scss file.